src/server/game/Chat/Channels/Channel.h

```diff
@@ -197,6 +201,7 @@ class Channel
         void UnBan(uint64 guid);
         void Password(Player const* player, std::string const& pass);
         void SetMode(Player const* player, std::string const& p2n, bool mod, bool set);
+        void SetOwnerGM(Player const* player, bool on);
```

Comment:
This function is not used, please remove.

Reply:
Why not leaving there as an extension of the function for future edits?
I don't think is does anything wrong

If you really want to delete it i will

Reply:
I would avoid adding unused code

Reply:
@wizzymore 
Please remove the function. It is generally bad practice to leave dead code in software projects. It just increases the complexity of the code and makes it harder to maintain the software.